jq_field_parser.go

```diff
@@ -7,14 +7,19 @@ import (
 	"github.com/flant/libjq-go/pkg/jq"
 )
 
+func appendEmptyExpr(expression string) string {
+	if expression == "" {
+		return "empty"
+	}
+	return expression + " // empty"
```

Comment:
Since we have changed this here, I want to also want to remove the `"null"` checks that are no longer relevant within this PR too.